src/guide/introduction.md

```diff
@@ -0,0 +1,87 @@
+# Iroha 2
+
+Iroha is a fully-featured
+[blockchain ledger](/guide/glossary.md#blockchain-ledgers). With Iroha you
+can:
+
+- Create and manage custom
+  [fungible assets](/guide/glossary.md#fungible-assets), such as currencies,
+  gold, and others
+- Create and manage [non-fungible](/guide/glossary.md#non-fungible-assets)
+  assets
+- Manage user accounts with a domain hierarchy and multi-signature
+  transactions
+- Use efficient portable smart contracts implemented either via
+  [WebAssembly](/guide/blockchain/wasm.md) or
+  [Iroha Special Instructions](/guide/blockchain/instructions.md)
+- Use both permissioned and permission-less blockchain deployments
```

Comment:
This intro (and the whole document) seems to be the first thing users see in the docs. If we will stick to having "Get Started" section, I think it should be there.